scripts/check-pkg-for-release.js

```diff
@@ -129,9 +129,9 @@ function checkPkgDependencyAddedInMDCPackage() {
 }
 
 function checkCSSDependencyAddedInMDCPackage() {
-  const name = pkg.name.split('/')[1];
+  const name = getPkgName();
   let nameMDC = pkg.name.replace('@material/', 'mdc-');
-  if (name === 'textfield') {
+  if (name === 'text-field') {
```

Comment:
I don't think we need this if statement at all anymore. 'mdc-' plus 'text-field' equals 'mdc-text-field'

Reply:
I removed the if condition and updated it to use `name` variable